### DISCUSSION TITLE
Some necessary features

### DISCUSSION THREAD

**Discussion by zareismail** [laravel/nova-issues#3289] [General]
I think nova needs a better UI/UX for a good experience. So, I offer the below features:

1. Add reload button into the table when errors happen
2. Add compare ability into the Trend metrics
3. Add the ability to determine that the actions and filters should be displayed as a group button or select element
4. Add the ability to use icon instead of text for actions (is good for inline actions)
5. Add the ability to determine that the inline actions should be displayed as a group button or select element 
6. Add responsiveness
7. Add wizard form
8. Add grid into the form
9. Add filter ability on metrics
10. Add ability to the form button to stay on the form after creation or update

Thanks

**Comment by vesper8** [comment]
I added a reload button in the top nav bar by adding it into `resources/views/vendor/nova/layout.blade.php`

```
<div class="mr-2 cursor-pointer">
    <svg onclick="Nova.$emit('refresh-resources')" class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 4v5h.582m15.356 2A8.001 8.001 0 004.582 9m0 0H9m11 11v-5h-.581m0 0a8.003 8.003 0 01-15.357-2m15.357 2H15"></path>
    </svg>
</div>
```

**Comment by vv-ac** [comment]
I can second on the following:
2. Add compare ability into the Trend metrics
4. Add the ability to use icon instead of text for actions (is good for inline actions)
6. Add responsiveness
7. Add wizard form
8. Add grid into the form
